src/common/strtol.cc

```diff
@@ -43,19 +55,16 @@ long long strict_strtoll(const char *str, int base, std::string *err)
     *err = errStr;
```

Comment:
It *looks* at first as if this is a bug (returning the value of a temporary), but it's ok because operator=() is overloaded and makes a copy. Still, I think it would be way more clear if you took the out-parameter by reference.